src/main/scala/io/github/kiemlicz/shelm/HelmPlugin.scala

```diff
@@ -172,7 +184,7 @@ object HelmPlugin extends AutoPlugin {
 
   private[this] def updateDependencies(chartDir: File, log: Logger): Unit = {
     log.info("Updating Helm Chart's dependencies")
-    retrying(s"helm dependency update $chartDir", log) // due to potential parallel runs...
+    retrying(s"helm dependency update $chartDir --skip-refresh", log) // due to potential parallel runs...
```

Comment:
this will prevent internal `helm repo update` before every `helm dependency update` and significantly speed up charts packaging

Reply:
Nice!